code/game/gamemodes/clock_cult/clock_helpers/scripture_checks.dm

```diff
@@ -43,7 +43,9 @@
 
 //changes construction value
 /proc/change_construction_value(amount)
-	GLOB.clockwork_construction_value += amount
+	if(!SSticker.current_state != GAME_STATE_PLAYING) //This is primarily so that structures added pre-roundstart don't contribute to construction value
```

Comment:
construction value doesn't do anything since the rework, though.

Reply:
I had some code for wall costs scaling with CV in, but it didn't work out; I must have missed this when I was gutting what I had before

Reply:
I do think I'll keep this in for future work, however; maybe even in this PR. It also makes more sense, and because CV is announced at round end, it gives players a picture of how large the base was.